NCTRC/frontend/screens/CovidInformationScreen.tsx

```diff
@@ -15,126 +19,153 @@ interface CovidInformationScreenProps {
                     onPress={() => {
-                        if (currentCap + 1 > maxCap) {
-                            navigation.navigate(AppScreens.MaxCap);
-                        } else if (traveled) {
-                            setIsolate(false);
-                            setGatherings(false);
-                            setTraveled(false);
-                            setContact(false);
-                            navigation.navigate(AppScreens.CovidError, {
-                                name: name,
-                                email: email,
-                                reason: 'Traveled in the last 30 days',
-                                concerns: concerns,
-                            });
-                        } else if (contact) {
-                            navigation.navigate(AppScreens.CovidError, {
-                                name: name,
-                                email: email,
-                                reason: 'Contact with person who tested positive for COVID-19',
-                                concerns: concerns,
-                            });
-                        } else if (gatherings) {
+        
+                        let userToCreate: components["schemas"]["UserRequestModel"] = {
+                                firstName: firstName, 
+                                lastName: lastName, 
+                                email: email
+                        }
+
+                        if(traveled == "yes") {
+                            yesQuestion = "Traveled in the past 14 days";
+                        } else if(symptoms == "yes") {
+                            yesQuestion = "Experienced or exposed to symptoms in the past 24 hours";
+                        } else if(covidTest == "yes") {
+                            yesQuestion = "Positive test or waiting for results";
+                        } else if(gatherings == "yes") {
+                            yesQuestion = "Attending events or gatherings";
+                        }
+                        
+                        if(yesQuestion != "") {
+                            console.log(yesQuestion);
+                            const exposedUser: components["schemas"]["SigninRequestModel"] = {
+                                user: userToCreate,
+                                signinData: { temperature: tempurature, yesQuestion: yesQuestion },
+                            };
+                            signinUser(exposedUser); 
                             navigation.navigate(AppScreens.CovidError, {
-                                name: name,
-                                email: email,
-                                reason: 'Attended large gathering',
-                                concerns: concerns,
+                                reason: yesQuestion,
                             });
-                        } else if (isolate) {
+                        } else if (tempurature >= 100.4) {
```

Comment:
Try to avoid magic numbers

Reply:
How would you check to see if someone has a temperature? Store 100.4 in a variable?

Reply:
yeah that's how I would do it. Not a super big deal, but this way people know what 100.4 means